test/basic.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+. ${top_srcdir}/test/test-common.sh
```

Comment:
Almost each time you're using variable substitution, you should quote it. Otherwise, if the variable contains spaces (e.g. a path with spaces), it will be expanded into two separate words, not a single argument. In this file alone you should do so in this line, lines 7, 8 (twice) and 16. The usual rule is: "If in doubt, quote." ;-)